bootstrap-master/values-bootstrap.yaml

```diff
@@ -0,0 +1,41 @@
+# bootstrap a master argocd instance
+bootstrap-project:
+  enabled: true
+  ci_cd_namespace: &ci_cd "master-argocd"
```

Comment:
ahhh right, i see what you're doing now.

Reply:
err, i think this is the right place for it .. the bootstrap and bootstrap-master charts are chart-of-charts .. the work is being done by the chart dependencies ...

yeah, you would struggle to use the bootstrap chart and cli overrides for this use case.

Reply:
Yeah I was more initially thinking about the master deploy - perhaps it's just helm deployed or something but yeah, this works too. And ovs this approach is GitOpsy and pluggable